src/guide/backporting.md

```diff
@@ -58,7 +58,17 @@ If your request to backport a change is denied, but you for some reason already
 
 ## Using the Skara tooling to help with backports
 
-The Skara tooling includes support for backports. [The official Skara documentation](https://wiki.openjdk.org/display/SKARA/Backports) describes in detail how to work with the tooling to create backport PRs on [GitHub](https://github.com) or using the CLI tools. As described in the documentation, the [`/backport`](https://wiki.openjdk.org/display/SKARA/Commit+Commands#CommitCommands-/backport) command can be used on a commit or a PR to create the backport PR. If a backport PR is manually created, set the PR title to `Backport <original commit hash>`. This ensures that the bots will recognize it as a backport as opposed to a main fix specifically targeting an older release. One can tell whether or not the bots recognized a PR as a backport by the [backport]{.label} label being added if it's recognized.
+The Skara tooling includes support for backports. [The official Skara documentation](https://wiki.openjdk.org/display/SKARA/Backports) describes in detail how to work with the tooling to create backport PRs on [GitHub](https://github.com) or using the CLI tools. As described in the documentation, the [`/backport`](https://wiki.openjdk.org/display/SKARA/Commit+Commands#CommitCommands-/backport) command can be used on a commit or a PR to create the backport PR:
+
+    /backport jdk21u
```

Comment:
Only the JDK 21 Updates Maintainers would backport to the `jdk21u` repository.  All other Committers/Reviewers would backport to `jdk21u-dev`.  Depending on the intended audience, you may want to update this example.

Reply:
Right. Yes, jdk21u-dev is a better example. Fixed.